cmd/clusterctl/client/repository/components_client_test.go

```diff
@@ -234,7 +234,7 @@ func Test_componentsClient_Get(t *testing.T) {
 			gs := NewWithT(t)
 
 			f := newComponentsClient(tt.fields.provider, tt.fields.repository, configClient)
-			got, err := f.Get(tt.args.version, tt.args.targetNamespace, tt.args.watchingNamespace)
```

Comment:
Changing this means that we don't have tests for `Get`. If we share the logic this might be fine, or we can add a test specifically for the `Get` without values? WDYT?

Reply:
how about I refactor the func to share the common code so I don't just duplicate the tests?